google-api-client-bom/pom.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0"?>
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
changes in this file don't appear connected to the PR

Reply:
The owlbot post processor made these chnages.